testCoverage.sh

```diff
@@ -57,17 +57,18 @@ done
 
 write_to_cov "### Backend" 
 
-cd ~/Documents/Projects/grafana/pkg/tsdb/cloudwatch/
+cd ${GRAFANA}/pkg/tsdb/cloudwatch/
     write_be_coverage 'CloudWatch'
-cd ~/Documents/Projects/grafana/pkg/tsdb/cloudmonitoring/
+cd ${GRAFANA}/pkg/tsdb/cloudmonitoring/
     write_be_coverage 'Cloud Monitoring'  
-cd ~/Documents/Projects/grafana/pkg/tsdb/azuremonitor/
+cd ${GRAFANA}/pkg/tsdb/azuremonitor/
     write_be_coverage 'Azure Monitor' 
 
 
 cd $PLUGINS;
 for d in */ ; do
     cd $d
+    cd pkg
```

Comment:
I'm wondering if you have any issues without going into `pkg` on your machine? Or if we have some different structure on our respective machines?

Reply:
I did not change anything to the structure... weird.

I have exact same results when running from root and from `pkg`